Packs/GoogleCloudLogging/Integrations/GoogleCloudLogging/GoogleCloudLogging.py

```diff
@@ -0,0 +1,285 @@
+            number_of_results_to_retrieve -= max_limit
+        else:
+            request_body |= ({'pageSize': number_of_results_to_retrieve, 'pageToken': next_page})
+            number_of_results_to_retrieve -= number_of_results_to_retrieve
+        demisto.debug(f'Request body: {request_body}')
+        next_response = client.get_entries_request(request_body)
+        next_page = next_response.get('nextPageToken')
+        entries.extend(next_response.get('entries', []))
+        next_response.update({'entries': entries})
+    return next_response
+
+
+def create_readable_output(response: list[dict]) -> str:
+    """
+    Gets readable output.
+
+    Args:
+        response (list[dict]): A List of logs entries.
+
+    Returns:
+        readable output string.
+    """
+    hr = [
+        {
+            'TimeStamp': entry.get('timestamp'),
+            'Log Name': entry.get('logName'),
+            'Insert ID': entry.get('insertId'),
+            'Principal Email': entry.get('protoPayload', {})
+            .get('authenticationInfo', {})
+            .get('principalEmail'),
+            'Type': entry.get('resource', {}).get('type', {}),
+            'Project ID': entry.get('resource', {})
+            .get('labels', {})
+            .get('project_id'),
+            'Cluster Name': entry.get('resource', {})
+            .get('labels', {})
+            .get('cluster_name'),
+            'Service Name': entry.get('serviceName'),
+        }
+        for entry in response
+    ]
+    return tableToMarkdown('Lists log entries', t=hr,
+                           headers=['TimeStamp', 'Log Name', 'Insert ID', 'Principal Email',
+                                    'Type', 'Project ID', 'Cluster Name', 'Service Name'],
+                           removeNull=True)
+
+
+def log_entries_list_command(client: GoogleCloudLoggingClient, args: Dict[str, Any]) -> CommandResults:
+    """
+    Gets lists log entries. Use this method to retrieve log entries that originated from a
+       project/folder/organization/billing account.
+
+    Args:
+        client (Client): A Client class to wrap the google cloud api library.
+        args (dict): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults containing the lists log entries.
+    """
+    resource_project_name = argToList(args.get('project_name', []))
+    resource_organization_name = argToList(args.get('organization_name', []))
+    resource_billing_account_name = argToList(args.get('billing_account_name', []))
+    resource_folders_names = argToList(args.get('folders_names', []))
+
+    if not (resource_project_name or resource_organization_name or resource_billing_account_name or resource_folders_names):
+        raise DemistoException('At least one resource from project_name, '
+                               'organization_name, billing_account_name, or folder_name must be provided.')
+    resources = []
+    for project_name in resource_project_name:
```

Comment:
Can I give 2 resources? or only one each time?
For example, can I give project and billing account?

Reply:
Also please check first what is given, no need to run on all if only one type is given.
For example:
for organization_name in resource_organization_name if resource_organization_name

Reply:
Yes, The user can provide 2 resources